pkg/static/login.scss

```diff
@@ -251,7 +251,7 @@ label.checkbox {
   display: inline-block;
 }
 
-.pf-v5-c-button[spinning] .spinner {
+.spinner {
```

Comment:
Thanks @marvinruder for fixing this! However, I think this breaks the other "spinner" in the "login-actions" form. That one is enabled/disabled dynamically:

pkg/static/login.js:        id("login-button").removeAttribute('spinning');
pkg/static/login.js:        id("login-button").setAttribute('spinning', "true");

To make this more targetted, I suppose this should keep the original selector, but add the new one:

```css
.pf-v5-c-button[spinning] .spinner,
#login-wait-validating .spinner {
```